src/lib/util/constants.ts

```diff
@@ -298,6 +300,11 @@ export const clientOptions: Partial<ClientOptions> = {
 	logger: {
 		instance: new Logger({ level: DEV ? LogLevel.Debug : LogLevel.Info })
 	},
+	api: {
+		auth: ({
+			transformers: [transformOauthGuildsAndUser]
```

Comment:
Because for some goddamn reason it creates a circular or something when in the config file causing everything to crash and burn